src/app/api/tools/cowswap/orderFlow.ts

```diff
@@ -93,7 +96,29 @@ export async function orderRequestFlow({
   const orderUid = await orderbook.sendOrder(order);
   console.log("Order Posted", orderbook.getOrderLink(orderUid));
 
+  const swapData: SwapFTData = {
+    network: {
+      name: chainId.toString(),
+      icon: "",
+    },
+    txnData: order,
+    type: "swap",
+    tokenIn: {
+      name: quoteRequest.sellToken,
```

Comment:
Is `tokenIn` really the `sellToken`? It could be perceived either way. I would rename this for clarity.